Contributing.md

```diff
@@ -0,0 +1,120 @@
+# PlanetaryPy Community Contributing Guide
+
+This document describes a very simple process suitable for activities
+in the Planetary Software Organization ecosystem, including the TC
+itself.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an
+	issue or pull request.
+* A **Committer** is a subset of contributors who have been given
+	write access to the repository.
+* The **Technical Committee (TC)** is a group of committers representing
+	the required technical expertise to resolve rare disputes.
+
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in
+doubt, log an issue, any additional policies about what to include
+will be provided in the responses. The only exception is a security
+disclosure which should be sent privately to the contact address
+listed in the [Code of Conduct](CoC).
+
+Committers may direct you to another repository, ask for additional
+clarifications, and add appropriate metadata before the issue is
+addressed.
+
+Please be courteous and respectful.  Every participant is expected
+to follow the [Code of Conduct](CoC)).
+
+# Contributions
+
+Any change to resources in this repository must be through pull
```

Comment:
same as above, what `repository` is meant here? The whole org? Or the core package?

Reply:
It means what it says: 'this repository,' the TC repo.  Again, each repo needs its own `Contributing` document.  I think they could all reference this one, be copied from it, or be logically descended from these contents.